gobblin-compaction/src/main/java/gobblin/compaction/dataset/TimeBasedSubDirDatasetsFinder.java

```diff
@@ -140,7 +136,7 @@ protected String getDatasetName(String path, String basePath) {
             try {
               folderTime = getFolderTime(jobInputPath, inputPath);
             } catch (RuntimeException e) {
-              log.warn(jobInputPath + " is not a valid folder. Will be skipped.");
+              log.warn(jobInputPath + " is not a valid folder. Will be skipped. Error: "+e);
```

Comment:
Try this: 
log.warn("{} is not a valid folder. Will be skipped due to exception.", jobInputPath, e);

http://www.slf4j.org/faq.html#paramException